tastyworks/models/greeks.py

```diff
@@ -0,0 +1,47 @@
+from dataclasses import dataclass
+from decimal import Decimal
+# from tastyworks.models.option import Option
+
+# [{'eventSymbol': '.SPY210419P410',
+#   'eventTime': 0,
+#   'eventFlags': 0,
+#   'index': 6951999236862902272,
+#   'time': 1618638457000,
+#   'sequence': 0,
+#   'price': 0.07332271,
+#   'volatility': 0.09516809,
+#   'delta': -0.04395782,
+#   'gamma': 0.02291552,
+#   'theta': -0.04936977,
+#   'rho': -0.00192595,
+#   'vega': 0.03967128}]
+
+
+@dataclass
+class Greeks(object):
```

Comment:
This looks very useful, thanks! Could we possibly do this change (adding the `Greeks` model) as a separate, simpler PR?

Reply:
I will see how to do that. I could not see a way to separate what files to submit for each PR.

Reply:
Just submitted a new PR just for the Greeks module